zephyr/docker-build.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright(c) 2021 Intel Corporation. All rights reserved.
+
+# "All problems can be solved by another level of indirection"
+# Ideally, this script would not be needed.
+
+set -e
+set -x
+
+# export http_proxy=...
+# export https_proxy=...
+
+unset ZEPHYR_BASE
+
+sudo --preserve-env=http_proxy,https_proxy apt-get update
+sudo --preserve-env=http_proxy,https_proxy apt-get -y install libssl-dev
```

Comment:
Submitted libssl-dev addition to the docker image in https://github.com/zephyrproject-rtos/docker-image/pull/58